api/token_metadata.cc

```diff
@@ -71,7 +71,7 @@ void set_token_metadata(http_context& ctx, routes& r, sharded<locator::shared_to
 
```

Comment:
Regarding the commit `token_metadata: rename endpoint_to_host_id_map getter ...`

If I understand correctly, this commit is fixing https://github.com/scylladb/scylladb/issues/17857
please confirm -- and if it's true, then add `Fixes: scylladb/scylladb#17857` at the end of the commit message. And also at the end of the PR description.